packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -892,6 +895,20 @@ class SlideshowView extends React.Component {
 
           this.props.actions.eventsListener(GALLERY_CONSTS.events.ITEM_ACTION_TRIGGERED, props, e);
         },
+        scrollToGroupIndex: (idx) => {
```

Comment:
When is this used? Please add a description on the PR.

Reply:
Adding Group Active Index and Group Navigation to the Panel API would be a fantastic enhancement! This feature could streamline navigation and provide developers with better control over grouped components[,](https://www.techbiti.com/snapdragon-profiler/) ensuring a smoother user experience and more organized UI management.